communities/aston-css/community-assessment.md

```diff
@@ -0,0 +1,159 @@
+## Assessing Your Community's Needs
+
+Fully understanding the needs of your community is a complex concept that will likely take lots of time, conversation, and observation.  Depending on your situation, you may already have a great idea for what your community needs. You may not, and that's OK. We will go over these questions and topics broadly during week 1, and discuss responses in more detail in week 2. Here are some things to keep in mind when assessing your community and hoping to grow it.
+
+Please write down your
+- Answers to these questions for current community
+- Answers to these questions for **an ideal version** of your current community
+- Ideas on how to improve in each category
+
+After you've completed these questions, define your community in several short paragraphs for discussion in week 2.
+
+### Definition
+Students at Aston University studying/interested in Computer Science or related subjects, and seeking help with the course or to learn more than is covered in their course.
+
+### Size
+
+- How many people are currently in your community?
+
+Around about 120 listed members (although not certain on this), but many do not regularly attend events. Probably about 30 semi-regulars.
```

Comment:
The data is actually available, I just need to go and march into the SU to get it 😅. We should probably also start recording attendance to events somewhere.